listeners.go

```diff
@@ -56,14 +60,19 @@ func handleTCPClient(conn net.Conn, f *os.File, ch *nbc.NonBlockingChan) {
 			return
 		}
 
+		log.Printf("New connection from %s to port %d -> glutton:Proxy\n", addr[0], dp)
+		counter.addConnection()
+
 		// Data Transfer between Connections
-		ProxyServer(conn.(*net.TCPConn), proxyConn, f)
+		clossedBy, err := ProxyServer(conn.(*net.TCPConn), proxyConn)
+		counter.connectionClosed(srcAddr, handler[6:], clossedBy, err)
 	}
-
 }
 
 // TCPListener listens for new TCP connections
-func TCPListener(f *os.File, ch *nbc.NonBlockingChan) {
+func TCPListener(ch *nbc.NonBlockingChan) {
+	C := Connections{}
```

Comment:
Sure, I will call it counters. I have just completed with UDP, I will update with changes.